src/main/resources/kibana-rest.yml

```
@@ -45,13 +50,15 @@ kibana_data_views:
 kibana_detection_engine_health_cluster:
   versions:
     ">= 8.8.2": "/internal/detection_engine/health/_cluster"
+    # ">= 9.0.0": "uri [/internal/detection_engine/health/_cluster] with method [get] exists but is not available with the current configuration"
```

Does this mean we should add `< 9.0.0` to the above line to disable it?

```suggestion
    ">= 8.8.2 < 9.0.0": "/internal/detection_engine/health/_cluster"
    # ">= 9.0.0": "uri [/internal/detection_engine/health/_cluster] with method [get] exists but is not available with the current configuration"
```